Introduction/Getting started/External Linter/task.md

```diff
@@ -1,13 +1,21 @@
-## Enable External Linter
+IntelliJ Rust plugin doesn't detect all the errors. It relies on the Rust compiler to do that. While you are learning Rust, it is useful to see errors as you type. To have this behavior, we recommend to enable an external linter as follows.
 
-IntelliJ Rust plugin doesn't detect all the errors. It relies on the Rust compiler to do that. While you are learning Rust, it is useful to see errors as you type. To have this behavior, we recommend to enable an external linter as follows:
+Once you do that, %IDE_NAME% will report all the errors IntelliJ Rust plugin or Rust compiler detect.
```

Comment:
Once you do that, %IDE_NAME% will report all errors detected by either the IntelliJ Rust plugin or the Rust compiler.